webapp/src/Controller/API/AbstractRestController.php

```diff
@@ -230,6 +230,8 @@ abstract protected function getIdField(): string;
 
     /**
      * @throws NonUniqueResultException
+     *
+     * @return Object[]
```

Comment:
We could use an `@template` at the class level here again and then have this `@return T[]`.